app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1> Here is a list of posts for a given user </h1>
\ No newline at end of file
+<section class="container">
+<article class="users-article">
+<div class="photo"><%= @user.photo %></div>
```

Comment:
- [ ] As suggested by the previous code reviewer to show the image of the user in all the views,  Kindly use img tag to show the image of the user in all the views including here 👍 

I also noticed that your users show page doesn't show anything:

image.png